ingest/validation/validate_metadata.py

```diff
@@ -827,7 +832,8 @@ def cast_metadata_type(metadatum, value, id_for_error_detail, convention, metada
         try:
             if "|" not in value:
                 msg = (
-                    f"There is only one array value, for {metadatum}: {value}. "
+                    f"{metadatum}: accepts an array of values.\n"
+                    "Lines detected with single values instead of arrays.\n"
```

Comment:
This message (which is only stored as a warning and currently isn't passed to the user unless they're using the CLI) used to report the line content. The problem is that the reporting may be more confusing than helpful because this warning is triggered whenenver a single ontologyID is provided where the data MIGHT be an array (it seemed onerous to force people to manually indicated an array of one item). Because arrays are so rarely used, the warning is largely a false positive so I'm honestly ambivalent about its existence except that we do need some way for people to get feedback that they should use pipes as delimiters if the want to give us an array (and used the wrong delimiter, like commas).